src/const.h

```diff
@@ -519,8 +519,13 @@ enum PlayerFlags : uint64_t {
 
 #define CHANNEL_GUILD 0x00
 #define CHANNEL_PARTY 0x01
+
+#define CHANNEL_CAST 0xFFFE
```

Comment:
What?
You have this problem?
Try to use #define CHANNEL_CAST 40
(without hexadecimal)